### PR TITLE
Update version: Nvidia.GeForceNow version 2.0.55.133

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/119189)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Standard error:  Installation failed with exit code -1978335226 
2023-09-08 11:41:09.486 [CLI ] ShellExecute installer failed: 3825205252

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335226 
2023-09-09 00:42:10.400 [CLI ] ShellExecute installer failed: 3825205252

File: manifests/n/Nvidia/GeForceNow/2.0.55.133/Nvidia.GeForceNow.locale.zh-CN.yaml [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
+- 串流
+- 云游戏
+- 游戏
+ReleaseNotesUrl: https://www.nvidia.com/en-us/geforce-now/release-highlights
+Documentations:
+- DocumentLabel: 常见问题
+  DocumentUrl: https://www.nvidia.com/en-us/geforce-now/faq
+- DocumentLabel: 快速入门指南
+  DocumentUrl: https://www.nvidia.com/content/dam/en-zz/Solutions/gfn/webassets/geforce-now-qsg-pc-mac.pdf
+ManifestType: locale
```
Comment (stephengillie) [inline manifests/n/Nvidia/GeForceNow/2.0.55.133/Nvidia.GeForceNow.locale.zh-CN.yaml:30]:
```suggestion
ManifestType: defaultLocale
```

**Comment by mdanish-kh** [comment]
Close with reason: No changes made to actual manifest fields;

**Pull request closed without merging**